hangman/lib/database.py

```diff
@@ -0,0 +1,19 @@
+import sqlite3
+
+
+contents = sqlite3.connect("hangman/data/store_guesses.sqlite")
+contents.execute("CREATE TABLE IF NOT EXISTS guesses(letter CHAR)")
+
+def guesses_table(guess):
```

Comment:
This function stores a guessed letter. Improve the naming here

Reply:
if i use guessed_letter, I'll have issues with naming when calling the file in core.py

Reply:
It's an action, you can include that in the name. `store_guessed_letter`

Reply:
ok